docs/sources/tempo/metrics-generator/service_graphs.md

```diff
@@ -43,6 +43,17 @@ Each emitted metrics series have the `client` and `server` label corresponding w
   tempo_service_graph_request_total{client="app", server="db", connection_type="database"} 20
 ```
 
+#### Virtual nodes
+
+Virtual nodes are nodes that form part of the lifecycle of a trace,
+but spans for them are not being collected because they're outside the user's reach (eg. an external service for payment processing) or are not instrumented (eg. a frontend application).
```

Comment:
```suggestion
but spans for them are not being collected because they're outside the user's reach (for example, an external service for payment processing) or are not instrumented (for example, a frontend application).
```